pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -156,6 +158,12 @@ func validateCommonServiceBrokerSpec(spec *sc.CommonServiceBrokerSpec, fldPath *
 		}
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(scfeatures.CatalogRestrictions) {
+		if spec.CatalogRestrictions != nil {
+			// TODO: nothing to check?
```

Comment:
If you want to validate the format of a restriction, here’s the place to do it

Reply:
I imagine we want to do some form of validation. How long can a restriction be?

Reply:
You should make sure each string parses here.

Reply:
If the feature gate isn’t enabled, having the restrictions field set should be invalid

Reply:
This is the last thing on the list to do and then I am "done"

Thanks for reminding me.